Puppetfile.stable

```diff
@@ -41,14 +41,9 @@ mod 'camptocamp-kmod',
   :git => 'https://github.com/simp/puppet-kmod.git',
   :tag => '2.2.0'
 
-# We are maintaining a simp-master branch of this module
-# because we needed changes earlier, and, due to a simp-adapter
-# bug WRT RPM upgrades with name changes, can't revert back
-# to camptocamp/systemd.  We will revert when we redesign module
-# RPM packaging (SIMP-6.4.0 or later).
 mod 'simp-systemd',
```

Comment:
please change from simp-systemd to camptocamp-systemd for clarity.  In Puppetfiles we list <owner>-<module name>